react/features/base/connection/actions.native.js

```diff
@@ -197,6 +198,13 @@ export function connect(id: ?string, password: ?string) {
                 (data: NativeXmppPostMethodEventData) => handlePostMethodEvent(data, stropheConn, dispatch)));
             nativeEventListeners.push(xmppBridgeEmitter.addListener(NativeEvents.XMPP_GET_METHOD,
                 (data: NativeXmppPostMethodEventData) => handleGetMethodEvent(data, stropheConn, dispatch)));
+            nativeEventListeners.push(xmppBridgeEmitter.addListener(NativeEvents.UPDATE_USER_AVATAR,
+                (jsonString: string) => {
+                    const { userXmppLoginId, imageUrl } = JSON.parse(jsonString);
```

Comment:
please do the json parse in the updateUserAvatar function instead of do in the lambda